### DISCUSSION TITLE
Support for iris rev8

### DISCUSSION THREAD

**Discussion by dliechty** [manna-harbour/miryoku#244] [Q&A] [answered]
I am brand new at this but I do not see a rev8 folder in the keyboards area. It stops at rev7. That being said, the "what's new" section on the keebio site says that the layout options are exactly the same as rev7, it's just a more powerful processor. Does this mean I can create a build for rev7 and it should work with no other changes?

[keebio iris](https://keeb.io/products/iris-keyboard-split-ergonomic-keyboard)

**Comment by manna-harbour** [comment, marked as answer]
QMK master is the current version of QMK, but usually does not contain the current version of Miryoku QMK. The Miryoku QMK development branch is the current version of Miryoku QMK, but usually does not contain the current version of QMK. To use both the current versions of QMK and Miryoku QMK together, the QMK master and miryoku branches need to be merged.
See https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#branches.

[rev8 is included in QMK master](https://github.com/qmk/qmk_firmware/tree/master/keyboards/keebio/iris/rev8). Here's a sample build run for rev8 merging QMK master: https://github.com/manna-harbour/miryoku_qmk/actions/runs/6198484881

At the moment, QMK master is probably up to date with Miryoku QMK, so if you're building locally you can just build directly from QMK master.

**Reply by vaporlabs** [reply]
I came looking for this same answer, but found that things have changed on `qmk_firmware` master. It appears the user keymaps were deleted in this commit: https://github.com/qmk/qmk_firmware/commit/1ed03f498fa204178c2696c510ac6a2cd8524e2d. I checked out the commit just before that https://github.com/qmk/qmk_firmware/commit/4908d4b1ca260efecf3613e6517aa3a6f2034876 and was able to build. What is the best way to do this going forward?

**Reply by manna-harbour** [reply]
@erroneousmaximus See https://github.com/manna-harbour/miryoku/discussions/287.

**Reply by manna-harbour** [reply]
@erroneousmaximus Here's a sample build run: https://github.com/manna-harbour/miryoku_qmk/actions/runs/7689134235.
